packages/api/test/e2e/send.e2e.ts

```diff
@@ -0,0 +1,23 @@
+import WebSocket from '@polkadot/x-ws';
+import { RpcCoder } from "@polkadot/rpc-provider/coder";
+
+describe('send', (): void => {
+
+    it('Send request using websocket', async (done)  => {
+        const providerUrl = 'wss://cennznet.unfrastructure.io/public/ws';
```

Comment:
if we test against azalea this test will tell us if @polkadot/rpc-provider update breaks with current mainnet or if mainnet breaks (which is too late at that point)

we should also test against the cennznet dev node here incase it breaks compatibility (the root cause of the sylo issue)